### PR TITLE
Fix Spelling and Grammar Mistakes in README.md

### PR DESCRIPTION
**Pull request by sundaram-sharma**
Hello,

I've made some changes to the README.md file to correct spelling and grammar mistakes to improve the overall readability and clarity of the document. These changes should make it easier for users to understand the project's instructions and purpose.

I've reviewed the content carefully, and I believe these corrections will enhance the quality of the documentation. Please let me know if you have any feedback or if there are additional changes you'd like me to make.

Thank you for considering this pull request!

Best regards,
Sundaram Sharma

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,24 +18,24 @@ their great cross-browser testing tools.
 
 ---
 
-English | [简体中文](./README-zh_CN.md)
+English | [简体中文](./README-zh_CN.md) | [Hindi](./README-in_HN)
```
Comment (jvanbruegge) [inline README.md:21]: There is no such file?

File: README.md [hunk 2]
```diff
@@ -171,7 +171,7 @@ extendable.
 
 The core exposes only one single function `init`. This `init`
 takes a list of modules and returns a `patch` function that uses the
-specified set of modules.
+the specified set of modules.
```
Comment (paldepind) [inline README.md:174]: There is an extra "the" here.

File: README.md [hunk 3]
```diff
@@ -312,7 +312,7 @@ desired points in the life of a virtual node.
 The following hooks are available for modules: `pre`, `create`,
 `update`, `destroy`, `remove`, `post`.
 
-The following hooks are available in the `hook` property of individual
+The following hooks are available in the `hook` property of the individual
```
Comment (paldepind) [inline README.md:315]: I don't think the "the" is appropriate here as the sentence is not referring to specific elements but just all elements in general?

**Comment by paldepind** [comment]
Thanks a lot for the PR @sundaram-sharma. This fixes a lot of grammar issues in the readme 👍 

Please look at the comments. There are two changes I don't agree with and perhaps you forgot to add the translated readme?

File: README.md (hunk 1 above)
Comment (sundaram-sharma) [reply README.md:21]: Hi @jvanbruegge , thanks for pointing this out. I just made another PR that contains (Feature) language support for Hindi, which contains this file.

**Pull request closed without merging**